DuggaSys/microservices/sharedMicroservices/retrieveUsername_ms.php

```diff
@@ -13,19 +13,18 @@
 pdoConnect();
 session_start();
```

Comment:
I'm not sure this is needed either, like `pdoConnect()`. Most likely the calling service will have performed this function call already.